src/core_plugins/kibana/public/dashboard/_index.scss

```diff
@@ -1,8 +1,3 @@
-// External libraries
```

Comment:
Can we add a comment just to let people know there is other styling at play and where to go to see it? Where it lives and where its imported would be enough.

Reply:
Done